src/Settings.js

```diff
@@ -23,6 +23,7 @@ function Settings(options) {
     this.displayMode = get(options.displayMode, false);
     this.throwOnError = get(options.throwOnError, true);
     this.errorColor = get(options.errorColor, "#cc0000");
+    this.macros = options.macros;
```

Comment:
Could you provide a default here so we don't have to check if `settings.macros` is defined when creating the `MacroExpander`?